test/puppetlabs/trapperkeeper/services/authorization/authorization_core_test.clj

```diff
@@ -0,0 +1,138 @@
+(ns puppetlabs.trapperkeeper.services.authorization.authorization-core-test
+  (:require [clojure.test :refer :all]
+            [puppetlabs.trapperkeeper.services.authorization.authorization-core :refer :all]
+            [schema.test]))
```

Comment:
Huh, cool, guess we don't have to always tack on the `:as schema.text` part.  So used to just always doing a few keywords at the end :)